src/app/shared/shared.module.ts

```diff
@@ -13,13 +13,15 @@ import { DragMoveDirective } from 'app/shared/utils/drag-move.directive';
 
 import { TableTemplateComponent } from 'app/shared/components/table-template/table-template.component';
 import { ListConverterPipe } from './pipes/list-converter.pipe';
+import { SafeHtmlPipe } from './pipes/safe-html-converter.pipe';
 import { OrgNamePipe } from './pipes/org-name.pipe';
 import { TableTemplateUtils } from './utils/table-template-utils';
 import { TableDirective } from './components/table-template/table.directive';
 import { PublishedPipe } from 'app/shared/pipes/published.pipe';
 import { Utils } from './utils/utils';
 import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
 import { NgZorroAntdModule, NZ_I18N, en_US } from 'ng-zorro-antd';
+import { from } from 'rxjs';
```

Comment:
Is this being uses somewhere below?

Reply:
No, I also don't remember importing that.  I can remove it if it's not being used there by something else already. I should check it in other branches I guess.

Reply:
Removed.